Harness-engineering-google-guide.html

```
@@ -0,0 +1,1630 @@
+        cursor: default;
+      }
+
+      footer.note {
+        margin-top: 40px;
+        color: var(--text-dimmer);
+        font-size: 13.5px;
+      }
+    </style>
+  </head>
+  <body>
+    <button class="nav-toggle" id="navToggle" aria-label="目次を開く">☰</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)